docs/src/main/paradox/user/lang/java/eventsourced.md

```diff
@@ -104,6 +104,6 @@ In the example below, we show a shopping cart that also has a checkout command.
 
 ## Registering the entity
 
-Once you've created your entity, you can register it with the @javadoc[`CloudState`](io.cloudstate.javasupport.CloudState) server, by invoking the `registerEventSourcedEntity` method. In addition to passing your entity class and service descriptor, you also need to pass any descriptors that you use for persisting events, for example, the `domain.proto` descriptor.
+Once you've created your entity, you can register it with the @javadoc[`Cloudstate`](io.cloudstate.javasupport.Cloudstate) server, by invoking the `registerEventSourcedEntity` method. In addition to passing your entity class and service descriptor, you also need to pass any descriptors that you use for persisting events, for example, the `domain.proto` descriptor.
```

Comment:
We have not renamed classes yet, right?

Reply:
Good point - my search/replace got carried away, will fix

Reply:
@michaelpnash Let me know when you are ready for this to be merged.